libraries/AP_HAL_ChibiOS/hwdef/PodmanH7/defaults.parm

```diff
@@ -0,0 +1,36 @@
+# Now setup the default battery pins driver analog pins and default
+SERIAL0_BAUD 460
+SERIAL1_BAUD 115
```

Comment:
You can do these with ```define DEFAULT_SERIALx_BAUD y``` etc in hwdef - please don't define the ones that have the right defaults